### ISSUE TITLE
The problem is with the project itself in this file commands/slash/invite.js

### ISSUE THREAD

**Issue by mohamedharby488-ops** [SudhanPlayz/Discord-MusicBot#888]
The problem is with the project itself in this file commands/slash/invite.js
at this code .setURL(https://discord.com/oauth2/authorize?client_id=$%7Bclient.config.clientId%7D&permissions=$%7Bclient.config.permissions%7D&scope=$%7Bclient.config.scopes.toString) ). replace(',', '%20')}))
        ]
    });
});
module.exports = command;

**Comment by JotaroKujo0525** [comment]
what exactly is the problem with that snippets of code?

**Comment by mohamedharby488-ops** [comment]
The problem is you, when I invite him to the bot via an invitation, he enters me. Invite me, this problem appears in the bot’s invitation
Invalid domain: Syndicate, Applications.commands
I'm doing the call again in the right way. Note that the problem is not in the bot. The bot is in this file with the commands / slash / calling.js
This is the wrong code in the file
 .setURL (https://discord.com/oauth2/authorize?client_id=$%7Bclient.config.clientId%7D&permissions=$%7Bclient.config.permissions%7D&scope=$%7Bclient.config.scopes.toString)). replace(',', '%20')}))
]
});
});
module.exports = command;
Please, correct the code in the project
In other words, the problem is, he said that the error in his statements is true.

**Comment by JotaroKujo0525** [comment]
Hello, there are a couple things that are wrong here
1. The code that is written in [invite.js](https://github.com/SudhanPlayz/Discord-MusicBot/blob/v5/commands/slash/invite.js) is `.setURL(`https://discord.com/oauth2/authorize?client_id=${client.config.clientId}&permissions=${client.config.permissions}&scope=${client.config.scopes.toString().replace(',', '%20')}`))`
The code you sent is
`.setURL (https://discord.com/oauth2/authorize?client_id=$%7Bclient.config.clientId%7D&permissions=$%7Bclient.config.permissions%7D&scope=$%7Bclient.config.scopes.toString)). replace(',', '%20')}))`

As you can see, there are a lot of things that got wrong in here. So idk if you edited the code and messed it up or smth happened with your code

**Comment by JotaroKujo0525** [comment]
Please replace the contents of your invite.js file with the contents of https://github.com/SudhanPlayz/Discord-MusicBot/blob/v5/commands/slash/invite.js

**Comment by DevAmiyo** [comment]
Fixed it in #889
